### ISSUE TITLE
 MONDO:0013907 bilateral generalized polymicrogyria

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#4527]
**Mondo term (ID and Label):**
 MONDO:0013907 bilateral generalized polymicrogyria

**Suggested new label:**
MICROCEPHALY, SHORT STATURE, AND POLYMICROGYRIA WITH OR WITHOUT SEIZURES ??

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**_Optional_: Any additional information (like supporting evidence, PubMed ID, etc.)**
The current preferred name from Mondo conflicts with https://hpo.jax.org/app/browse/term/HP:0032410 "Bilateral generalized polymicrogyria" but Mondo does not assert this term maps as similar to the HPO term. 
I would suggest using the preferred name from OMIM for this entry (many "related" synonyms are OMIM names) But I am having a hard time telling if this term or the parent term (MONDO:0018764 , microcephalic primordial dwarfism due to RTTN deficiency) is intended as the exact match the MIM entry? If the term from GARD (https://rarediseases.info.nih.gov/diseases/10786/bilateral-generalized-polymicrogyria) and the Orphanet terms on  MONDO:0013907 aren't an exact match the OMIM entry, then maybe this should be restructured so the GARD-linked concept is the parent (broader) concept instead of the child?

**Comment by nicolevasilevsky** [comment]
@kanems I think OMIM:614833 is equiv to microcephalic primordial dwarfism due to RTTN deficiency, I moved this xef.
